Tasks/AzurePowerShellV4/InitializeAz.ps1

```diff
@@ -0,0 +1,63 @@
+[CmdletBinding()]
+param
+(
+    [String] [Parameter(Mandatory = $true)]
+    $endpoint,
+
+    [String] [Parameter(Mandatory = $false)]
+    $targetAzurePs
+)
+
+$endpointObject =  ConvertFrom-Json  $endpoint
+$moduleName = "Az.Accounts"
+$environmentName = $endpointObject.environment
+
+if($targetAzurePs -eq ""){
+    $module = Get-Module -Name $moduleName -ListAvailable | Sort-Object Version -Descending | Select-Object -First 1
+}
+else{
+    $module = Get-Module -Name $moduleName -ListAvailable | Where-Object {$_.Version -eq $targetAzurePs} | Select-Object -First 1
+}
+      
+if (!$module) {
+    # Will handle localization later
+    Write-Verbose "No module found with name: $moduleName"
+    throw ("Could not find the module Az.Accounts with given version. If the module was recently installed, retry after restarting the Azure Pipelines task agent.")
+}
+
+# Import the module.
+Write-Host "##[command]Import-Module -Name $($module.Path) -Global"
+$module = Import-Module -Name $module.Path -Global -PassThru -Force
+
+# Clear context
```

Comment:
Add command that is equivalent to `Disable-AzureRmContextAutosave`

Reply:
It is done